index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <script src="https://kit.fontawesome.com/045dcd1183.js" crossorigin="anonymous"></script>
+  <title>Jeff Kayombo- Portfolio</title>
+</head>
+<body>
+  <nav>
+    <div class="logo">
+      <h2><a href="#">Welcome</a></h2>
+    </div>
+    <div class="nav-menu">
+      <i class="fas fa-bars"></i>
+    </div>
+  </nav>
+  <section class="background-cover">
+    <header class="home-text">
+      <p>Hey there. I’m<br>Jeff<br><span>I’m a software<br>developer</span></p>
+    </header>
+    <header class="welcome-txt">
+      <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    </header>
+    <div class="social-icons">
+      <i class="fab fa-twitter"></i>
+      <i class="fab fa-linkedin-in"></i>
+      <i class="fab fa-medium-m"></i>
+      <i class="fab fa-github"></i>
+      <i class="fab fa-angellist"></i>
```

Comment:
- [ ] <b>REQUIRED:</b> Project requires that these are supposed to be in the list with `ul` tag and `li` tad per item. kindly fix that.